Netlify CMS

tomsabin opened the pull request:
Work in progress PR to have a Netlify deploy preview built, to test and explore the options. I am investigating Netlify CMS as an alternative to Prose, mainly due to an issue with file extensions produced in Prose https://github.com/middleman/middleman/issues/1211 (which seems to come from the [hardcoded placeholder filename](https://github.com/prose/prose/blob/869b40b552da9d9915e75b7abee68b2bfd307c7e/app/models/file.js#L19) since upgrading to Middleman v4 when ["Implied Extensions" feature was removed](https://github.com/middleman/middleman/commit/ce7636ad18340a41368705c06ecfc6dbab455ebe))

Commit:
Add Netlify CMS admin page and config

ubxd commented:
Deploy preview for *unboxed-co-production* ready!

Built with commit 7affd34bac31486b268180868874f548b861f0bf

https://deploy-preview-453--unboxed-co-production.netlify.com

Commit:
Specify Netfliy CMS config file to allow load from /admin and /admin/

To boot the Netlify CMS app, the URL that we visit must be /admin/.
However, the first time using the app, it was not obvious that the
trailing forward slash was required, and I dug into why the route didn't
work (thinking it was an issue with my config.yml - validation, syntax,
etc.)

However, we can get around this confusion by providing the config file
as a 'cms-config-url' link type in the admin/index.html page. Now we can
use /admin or /admin/, both of which will boot the Netlify CMS app.

Commit:
Add blog post fields to collection

Commit:
Make tags field optional

Commit:
Add weekly roundup collection

Commit:
Include application.css to render site styles in preview pane

Commit:
Add news articles collection

Commit:
Add additional button to body widget (rich text) for external images
